pkg/option/config.go

```diff
@@ -2855,6 +2861,7 @@ func (c *DaemonConfig) Populate() {
 	c.KeepConfig = viper.GetBool(KeepConfig)
 	c.KVStore = viper.GetString(KVStore)
 	c.KVstoreLeaseTTL = viper.GetDuration(KVstoreLeaseTTL)
+	c.CNIChainingMode = viper.GetString(CNIChainingMode)
```

Comment:
Nit: the original commit had this sitting between `ClusterMeshConfig` and `DatapathMode`.